### ISSUE TITLE
allow account creation with username/password combination

### ISSUE THREAD

**Issue by pbielefeldt** [ruuvi/com.ruuvi.station#1345]
Several people live in our household, and we all like to interact with the Ruuvi Sensors we have through the Ruuvi app (Android). 

I would generally like to take ownership of the Sensors to an account. However, as far as I understand, the only way to authenticate with the accounts is one-time token sent to an email address. There is no email account that all people in our household share. If there was a way to authenticate with a username (could be an email address of course) plus password combination, we could easily share the credentials with all people that I want to gain access to the account. (I understand that this is not ideal from a security point of view, but I would be fine with that in this case.)

To the best of my knowledge, there is no way to share an account in such a way right now. Am I overlooking something?

**Comment by markoaamunkajo** [comment]
Yes, correct. There is only ability to share through Ruuvi Cloud and this requires using email address, it's a security detail.
